protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -368,6 +368,26 @@ message DeleteTagRequest {
     }
 }
 
+
+message MergeRepositoryCommits {
+    RepositoryIdentification repository_id = 1;
+    string commit_sha_a = 2;
+    string commit_sha_b = 3;
+
+    CommitContent content = 4;
+    //repeated BlobExpanded conflict_resolution_blobs = 5; // Set of blobs that must resolve any conflict encountered
+    //bool force_manual_conflict_resolution = 6;
+
+    message Response {
+        Commit commit = 1;
+
+        // In case of a conflict, stop at the first such conflict and return it
+        //Commit common_base = 2; // Common base shared between the two commits
+        //repeated BlobDiff diffs_a = 3; // Diff between the common base and commit A
+        //repeated BlobDiff diffs_b = 4; // Diff between the common base and commit B
+    }
+}
```

Comment:
Already merged by me before to create the proxy.

Reply:
yes but had to do to compile. The diff will go once we merge master to the feature branch and feature branch to my branch.